example/20-componentconfig-gardener-controller-manager.yaml

```diff
@@ -29,6 +29,9 @@ controllers:
     staleGracePeriodDays: 14
     staleExpirationTimeDays: 90
     staleSyncPeriod: 12h
+  event:
+    concurrentSyncs: 5
+    ttlShortLivedEvents: 4h
```

Comment:
Why is it `4h` here while to default to `1h` in the `config/v1alpha1/defaults.go`?

Reply:
1h is the default ttl if none is configured in the kube-api-server.
4h is what we currently have configured in the landscape-setup https://github.wdf.sap.corp/kubernetes/landscape-setup/pull/1466/files
What should we chose? `ttlShortLivedEvents` should replace the ttl currently configured in the kube-api-server. Should I assume the k8s default of 1h our 4h? I lean towards 1h and would update this in all places. WDYT?

Reply:
I'm fine with `4h`, but it should be the same in both places (defaults in the config API and defaults in the example files/Helm chart values).

Reply:
As discussed going to default it to 1h. But making it consistent at all places